### PR TITLE
Fix comment in hashtag example of streams quickstart doc

### PR DESCRIPTION
**Pull request by pm47**

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @pm47,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

File: akka-docs/src/test/scala/docs/stream/TwitterStreamQuickstartDocSpec.scala [hunk 1]
```diff
@@ -155,7 +155,7 @@ class TwitterStreamQuickstartDocSpec extends AkkaSpec {
       .filterNot(_.hashtags.contains(akkaTag)) // Remove all tweets containing #akka hashtag
       .map(_.hashtags) // Get all sets of hashtags ...
       .reduce(_ ++ _) // ... and reduce them to a single set, removing duplicates across all tweets
-      .mapConcat(identity) // Flatten the stream of tweets to a stream of hashtags
+      .mapConcat(identity) // Expand the set of hashtags to a stream of hashtags
```
Comment (raboof) [inline akka-docs/src/test/scala/docs/stream/TwitterStreamQuickstartDocSpec.scala]: I kind of liked 'flatten' here, but otherwise :+1:

**Review by raboof — APPROVED** [review]

**Comment by raboof** [comment]
OK TO TEST

File: akka-docs/src/test/scala/docs/stream/TwitterStreamQuickstartDocSpec.scala (hunk 1 above)
Comment (patriknw) [reply akka-docs/src/test/scala/docs/stream/TwitterStreamQuickstartDocSpec.scala]: @pm47 could you change back to Flatten here

Comment (pm47) [reply akka-docs/src/test/scala/docs/stream/TwitterStreamQuickstartDocSpec.scala]: @patriknw done

**Review by patriknw — APPROVED** [review]

**Pull request merged**